tests/_support/Traits/With_Test_Container.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace StellarWP\Telemetry\Tests\Support\Traits;
+
+use StellarWP\Telemetry\Config;
+use StellarWP\Telemetry\Core;
+use StellarWP\Telemetry\Tests\Container;
+
+trait With_Test_Container {
+	/**
+	 * @before
+	 */
+	public function set_container(): void {
+		$test_container = new Container();
+		$test_container->setVar( Core::SITE_PLUGIN_DIR, '/app/public/wp-content/plugins/' );
```

Comment:
Is this specific to Slic?

Reply:
No, `slic` WordPress path is `/var/www/html`; this is just to set the value to _something_ and avoid complaints from the code.